server/webapp/WEB-INF/applicationContext-acegi-security.xml

```diff
@@ -263,6 +266,8 @@
                 /api/admin/material_test=ROLE_SUPERVISOR,ROLE_GROUP_SUPERVISOR
                 /api/admin/internal/pipelines=ROLE_SUPERVISOR,ROLE_GROUP_SUPERVISOR
                 /api/admin/**=ROLE_SUPERVISOR
+                /api/config-repository/**=ROLE_SUPERVISOR
+                /api/config-repository.git/**=ROLE_SUPERVISOR
```

Comment:
It conflicts with [this rule](https://github.com/gocd/gocd/blob/453f3d60368484f094ca0e6633d060a6042ba1f9/server/webapp/WEB-INF/urlrewrite.xml#L93)